### ISSUE TITLE
EasyLOB, an Archetype for LOB Applications

### ISSUE THREAD

**Issue by EasyLOB** [MarcosMeli/FileHelpers#214]
Thanks for creating FileHelpers!
I just published EasyLOB, an Open Source Archetype for building ASP.NET MVC LOB Application, at GitHub:

https://github.com/EasyLOB/EasyLOB/wiki

File Helpers is the library I recommend to read and write text files using EasyLOB.

**Comment by mcavigelli** [comment]
Thank you for considering and recommending the FileHelpers library.
I wanted to find out, how you are using it.
I have tried the github's repository search. The search did not find any occurence of 'FileHelpers'.
Are you using it or just recommending it?

Being curious, Matthias

**Comment by EasyLOB** [comment]
Hi Matthias,

 

I develop Line of Business Applications for almost 25 years, and using .NET technnology for almost 13 years.

EasyLOB is my current attempt ( and first Open Source ) to put together all tools and libraries you would need to develop them.

Even today developing a stable and scalable LOB Application is a mess in most of the places.

This kind of Application usually lasts for years and so choosing the appropriate Architecture and Tools is mandatory.

 

I have used your FileHelpers Library in many Web Forms Applications I created for years, to export and import all sorts off TXT files: it is specially usefull in Master-Detail TXT Files.

It is, by far, the best TXT Files library: that's why I recommended it.

You find an example of FileHelpers in the following links:

 

EasyLOB 1

https://github.com/EasyLOB/EasyLOB-Chinook/blob/master/Chinook.Application/Chinook/ChinookApplication.TXT.cs

 

EasyLOB 2

https://github.com/EasyLOB/EasyLOB-Chinook-2/blob/master/Chinook.Application/ChinookApplication/ChinookApplication.TXT.cs

 

It's a very simple example compared to what I have already done with FileHelpers.

But I included it, and recommended FileHelpers, because it's a MUST for every "serious" developer.

Even after XML, SOAP Web Services and REST APIs, TXT Files very alive in IT, at least in Brazil, where I live.

 

So I just can say “Thank You” for developing FileHelpers and for making it Open Source.

 

Siegmar Gieseler

 

De: Matthias Cavigelli [mailto:notifications@github.com] 
Enviada em: quinta-feira, 11 de maio de 2017 16:28
Para: MarcosMeli/FileHelpers <FileHelpers@noreply.github.com>
Cc: EasyLOB <easylob@gmail.com>; Author <author@noreply.github.com>
Assunto: Re: [MarcosMeli/FileHelpers] EasyLOB, an Archetype for LOB Applications (#214)

 

Thank you for considering and recommending the FileHelpers library.
I wanted to find out, how you are using it.
I have tried the github's repository search. The search did not find any occurence of 'FileHelpers'.
Are you using it or just recommending it?

Being curious, Matthias
